src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1597,7 +1597,7 @@ Please specify one of them using the --source option to proceed.</value>
     <value>Version to which to pin the package. The wildcard '*' can be used as the last version part</value>
   </data>
   <data name="PinAddBlockingArgumentDescription" xml:space="preserve">
-    <value>Block from upgrading until the pin is removed, preventing override arguments</value>
+    <value>Block from upgrading until the pin is removed, preventing override arguments.</value>

```

Comment:
Why only here and not the next line, and every other description?

Reply:
For the context: All the ones with name ending in `ArgumentDescription` are what is shown when you do `--help` for a command. Not sure if we should have a period or not in those, but it should be consistent since it's a pretty well defined category